app/audit/models/ViewNextOutstandingVatPaymentAuditModel.scala

```diff
@@ -17,17 +17,20 @@
 package audit.models
 
 import models.User
-import models.payments.Payment
+import models.payments.Payments
 
 case class ViewNextOutstandingVatPaymentAuditModel(user: User,
-                                                   payment: Option[Payment]) extends AuditModel {
+                                                   payments: Option[Payments]) extends AuditModel {
 
-  private val paymentDetails: Map[String, String] = payment match {
-    case Some(data) => Map(
+  private val paymentDetails: Map[String, String] = payments match {
+    case Some(data) if data.financialTransactions.size == 1 => Map(
```

Comment:
This if statement isn't needed. If the case below is moved above you can assume if it fails that `if data.financialTransactions.size > 1` then the size will always be 1. 

You can look at the ViewNextOpenVatObligationsAuditModel.scala for an example of this